app/src/main/java/com/example/stromtracker/ui/geraete/GeraeteFragment.kt

```diff
@@ -200,20 +199,26 @@ class GeraeteFragment : Fragment(), View.OnClickListener {
                 buttonSortName.paintFlags = Paint.UNDERLINE_TEXT_FLAG
                 buttonSortVerbrauch.paintFlags = 0
                 buttonSortRaum.paintFlags = 0
+                buttonSortRaum.typeface = Typeface.DEFAULT_BOLD
+                buttonSortName.typeface = Typeface.DEFAULT_BOLD
+                buttonSortVerbrauch.typeface = Typeface.DEFAULT_BOLD
```

Comment:
Ggf. könntest du das unterstreichen und fett markieren in eine Funktion auslagern, z.B. so:
...
setFirstSelected(buttonSortVerbrauch, buttonSortName, buttonSortRaum)
...
fun setFirstSelected(sel : Button, btn2 : Button, btn3 : Button) {
sel.paintFlags = Paint.UNDERLINE_TEXT_FLAG
btn2.paintFlags = 0
btn3.paintFlags = 0
//ggf noch
sel.typeface = Typeface.DEFAULT_BOLD
...
}

Reply:
Wäre eine Möglichkeit, ich wollte aber erst noch schauen ob es eine schönere Lösung gibt.
Falls wir nichts finden, dann können wir das auf jeden Fall machen.